### DISCUSSION TITLE
User bound Items

### DISCUSSION THREAD

**Discussion by CallMeJuRa** [Pryaxis/TShock#2995] [Q&A] [answered]
Is there any way to have ur items locked on your account?
So in the beginning you have to use /register
And with SSC all your progress will saved on the server.
If you now login from a different computer BUT use the same server account, is there any chance to get ur old items?

Would be awesome for Crossplay, so people can have their items on mobile and PC

**Comment by sgkoishi** [comment, marked as answer]
With SSC you will get the same progress once you login to the same account, no matter what device you are using. However, due to the client-side limitation, it is not possible to get items [favourited](https://terraria.wiki.gg/wiki/Favourite).
